lib/migration-chunks/validation/errors.js

```diff
@@ -31,6 +31,20 @@ module.exports = {
       CONTENT_TYPE_DOES_NOT_EXIST: (id, ctId) => {
         return `You cannot set a property on field with id "${id}" on content type "${ctId}" because it does not exist.`;
       }
+    },
+    move: {
+      FIELD_DELETED: (id) => {
+        return `You cannot move the field with id "${id}" because it has already been deleted.`;
+      },
+      FIELD_DOES_NOT_EXIST: (id) => {
+        return `You cannot move the field with id "${id}" because it does not exist.`;
+      },
+      FIELD_ALREADY_MOVED: (id) => {
+        return `You cannot move the field with id "${id}" more than once per migration.`;
+      },
+      CONTENT_TYPE_DOES_NOT_EXIST: (id, ctId) => {
+        return `You cannot move a field on content type "${ctId}" because it does not exist.`;
+      }
     }
```

Comment:
do we check if users are trying to move a field before or after a field which does not exist?

Reply:
:10x: catch. Fixed.